alembic/versions/a31c98c74ea0_add_e_p_v.py

```diff
@@ -34,6 +34,7 @@ def upgrade():
             {'id': 4, 'name': 'pypi', '_backend': 'pypi', 'url': 'https://pypi.python.org/', 'fetch_url': 'https://pypi.python.org/pypi'},
             {'id': 5, 'name': 'go', '_backend': 'scm', 'url': None, 'fetch_url': None},
             {'id': 6, 'name': 'crates', '_backend': 'crates', 'url': 'https://crates.io/', 'fetch_url': None},
+            {'id': 7, 'name': 'nuget', '_backend': 'nuget', 'url': 'https://nuget.org/', 'fetch_url': 'https://api.nuget.org/packages/'},
```

Comment:
Yes, the https://api.nuget.org/packages/ itself does not represent any resource, but we use it as base uri for fetching nuget artefacts, like:

https://api.nuget.org/packages/nunit.3.7.1.nupkg
https://api.nuget.org/packages/newtonsoft.json.10.0.3.nupkg